components/nexus-core/src/main/java/org/sonatype/nexus/maven/tasks/descriptors/ReleaseRemovalTaskDescriptor.java

```diff
@@ -48,7 +51,9 @@
           NUMBER_OF_VERSIONS_TO_KEEP_FIELD_ID, "Number to keep", "The number of versions for each GA to keep",
           FormField.MANDATORY),
       new RepoTargetComboFormField(REPOSITORY_TARGET_FIELD_ID, "Repository Target",
-          "Select a repository target to apply", FormField.OPTIONAL)
+          "Select a repository target to apply", FormField.OPTIONAL),
+      new CheckboxFormField(INDEX_BACKEND, "Use index",
+          "Removal should happen by crawling the repository (unchecked) or by using Maven Index?", FormField.MANDATORY)
```

Comment:
Could reword to: `Use Maven Index to walk the repository? Defaults to crawling if unchecked`